string uxv_name
string target_name
bool is_near_hospital
